Source/CarthageKit/Xcode.swift

```diff
@@ -1200,6 +1200,9 @@ private func binaryURL(_ packageURL: URL) -> Result<URL, CarthageError> {
 		}
 
 	default:
+		if let binaryURL = bundle?.executableURL {
+			return .success(binaryURL)
+		}
```

Comment:
It doesn't seem like this should be necessary because AIUI a static framework should be a `.framework` (which is why we also have a `MachOType` and `FrameworkType`).

Either way this should be tested as well.